internal/route/routes.go

```diff
@@ -45,16 +46,17 @@ func HandleIndex(w http.ResponseWriter, r *http.Request) {
 			// When I sanitize the string before splitting
 			// it doesn't work, so for now I'm just sanitizing
 			// each line
-			t = p.Sanitize(strings.TrimSpace(t))
-			todos = append(todos, t)
+			todoList.Todos = append(todoList.Todos, &model.Todo{
+				Todo: p.Sanitize(strings.TrimSpace(t)),
+			})
 		}
 
-		insertedTodos, err := model.CreateTodos(listName, todos)
+		err := todoList.Save()
```

Comment:
So now we just call `Save()` on that new/fresh `TodoList` we made at the top. It'll persist and fill in the missing fields from what the DB generated (namely the uuid id).